turbo.json

```diff
@@ -1,22 +1,14 @@
 {
   "$schema": "https://turborepo.org/schema.json",
-  "baseBranch": "origin/master",
   "pipeline": {
     "build": {
       "dependsOn": [
         "^build"
       ],
       "outputs": [
-        "dist/**",
-        "api/**",
-        "helpers/**",
-        "loaders/**",
-        "migrations/**",
-        "models/**",
-        "repositories/**",
-        "services/**",
-        "subscribers/**",
-        "utils/**"
+        "!node_modules/**",
+        "!src/**",
```

Comment:
suggestion: should we add `!./**` and `!.*`? to exclude all .XXX files and directory? or is it taking into account automatically?

Reply:
These won't be included unless they were declared in a sub-directory of a package. For example, the following `packages/foo/src/.bar` would be picked up, but we don't expect such cases. Even so, it's not a big deal to have them in the cache.

That being said, better safe that sorry. So I'll work on including your suggestion but I'll keep that for a future iteration.